containers-int-strict/src/Map/Int.hs

```diff
@@ -34,10 +34,10 @@ module Map.Int
        , alter
        ) where
 
-import Prelude hiding (lookup, null)
+import Control.DeepSeq (NFData (..))
 import Data.Coerce (coerce)
-
 import qualified Data.IntMap.Strict as M
+import Prelude hiding (lookup, null)
 
 newtype Map k v = IM (M.IntMap v)
     deriving newtype (Show, Eq)
```

Comment:
Instead of writing `NFData` instance manually you can add it to this list in `deriving newtype`